src/acceptance-test/java/net/consensys/orion/acceptance/oracle/OracleSendReceiveTest.java

```diff
@@ -0,0 +1,119 @@
+import net.consensys.orion.cmd.Orion;
+import net.consensys.orion.config.Config;
+
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpClient;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(TempDirectoryExtension.class)
+class OracleSendReceiveTest {
+
+  private static final String PK_2_B_64 = "Ko2bVqD+nNlNYL5EE7y3IdOnviftjiizpjRt+HTuFBs=";
+  private static final String HOST_NAME = "127.0.0.1";
+  private static final String JDBC_URL = "jdbc:oracle:thin:%s/%s@%s:%s:%s";
+
+  private Orion orionLauncher;
+  private Vertx vertx;
+  private HttpClient httpClient;
+  private final String databaseUser = System.getenv("ORACLE_USER");
+  private final String databasePassword = System.getenv("ORACLE_PASSWORD");
+  private final String databaseHost = System.getenv("ORACLE_HOST");
+  private final String databasePort = System.getenv("ORACLE_PORT");
+  private final String databaseSid = System.getenv("ORACLE_SID");
+
+
+  @BeforeEach
+  void setUp(@TempDirectory Path tempDir) throws Exception {
+    Assumptions.assumeTrue(databaseUser != null && databasePassword != null, "Oracle DB not configured");
+
+    final String jdbcUrl =
+        String.format(JDBC_URL, databaseUser, databasePassword, databaseHost, databasePort, databaseSid);
+
+    try (Connection conn = DriverManager.getConnection(jdbcUrl)) {
+      final Statement st = conn.createStatement();
+      st.executeUpdate(
+          " BEGIN\n"
+              + "      EXECUTE IMMEDIATE 'DROP TABLE store';\n"
+              + "  EXCEPTION\n"
+              + "      WHEN OTHERS THEN NULL;\n"
+              + "  END;");
+      st.executeUpdate("CREATE TABLE store(key varchar2(120) primary key, value BLOB)");
+    }
+
+    Path key1pub = copyResource("key1.pub", tempDir.resolve("key1.pub"));
+    Path key1key = copyResource("key1.key", tempDir.resolve("key1.key"));
+    Path key2pub = copyResource("key2.pub", tempDir.resolve("key2.pub"));
+    Path key2key = copyResource("key2.key", tempDir.resolve("key2.key"));
+
+    final Config config = NodeUtils.nodeConfig(
+        tempDir,
+        0,
+        HOST_NAME,
+        0,
+        HOST_NAME,
+        "node1",
+        joinPathsAsTomlListEntry(key1pub, key2pub),
+        joinPathsAsTomlListEntry(key1key, key2key),
+        "off",
+        "tofu",
+        "tofu",
+        "sql:" + jdbcUrl);
+
+    vertx = vertx();
+    orionLauncher = NodeUtils.startOrion(config);
+    httpClient = vertx.createHttpClient();
+  }
+
+  @AfterEach
+  void tearDown() {
+    orionLauncher.stop();
+    vertx.close();
+  }
+
+  @Test
+  @Disabled("Oracle DB needs to be available for this test.")
```

Comment:
There's a Assumption.assumeTrue in the setup so this doesn't need to be disabled, it will only run if the Oracle environment variables are defined

Reply:
Yes, you are right, but I kind of liked making this obvious and not dependent on an environment variable being set. At the moment the only way to run this test is if you set the DB up manually. 
The result in the test reports is the same (ignored) but I feel it is good to show someone that this cannot work. Maybe the message should state that licensing needs to be worked out so that an Oracle DB can be made available for the AT. 
What do you think?

Reply:
I think that the assumption is the better way to go as this can work and It's just a matter of adding the ojdbc jar to the classpath and setting up an Oracle database. With this disabled it's indicating that this test isn't expected to work which isn't this case, it just takes some setup.

Reply:
I think that the assumption is the better way to go as this can work and It's just a matter of adding the ojdbc jar to the classpath and setting up an Oracle database. With this disabled it's indicating that this test isn't expected to work which isn't this case, it just takes some setup.

Reply:
OK, I'll remove it!